modules/cats-effect-io/src/main/scala/io/janstenpickle/trace4cats/cats/effect/ProvideForIOLocal.scala

```diff
@@ -0,0 +1,21 @@
+package io.janstenpickle.trace4cats.cats.effect
+
+import cats.arrow.FunctionK
+import cats.{~>, Monad}
+import cats.effect.kernel.Resource
+import cats.effect.{IO, IOLocal}
+import io.janstenpickle.trace4cats.base.context.Provide
+
+private[trace4cats] class ProvideForIOLocal[Ctx](rootCtx: IOLocal[Ctx]) extends Provide[IO, IO, Ctx] {
```

Comment:
@mrdziuban We need just one implementation, no more need for `IdLift`/`IdUnlift` mixins.